src/Chanterelle/Internal/Utils/Error.purs

```diff
@@ -0,0 +1,102 @@
+module Chanterelle.Internal.Utils.Error where
```

Comment:
Some of this functions are not used and it seams like some of them don't quite pass the [Fairbairn threshold](https://wiki.haskell.org/Fairbairn_threshold).

Reply:
I would also agree with this